src/modules/user/beneficiaries/sagas/beneficiariesSaga.ts

```diff
@@ -9,7 +10,12 @@ const config: RequestOptions = {
 
 export function* beneficiariesSaga(action: BeneficiariesFetch) {
     try {
-        const beneficiaries = yield call(API.get(config), '/account/beneficiaries');
+        let params = '';
+        if (action.payload) {
+            params = `?${buildQueryString(action.payload)}`;
+        }
```

Comment:
Can you modify buildQueryString function by next way:
- If the argument of this function is empty or undefined or there is nothing to return -> return '';
- If there is some value to return, then return value with question mark -> '?<built value>';

Reply:
let me fix this in another PR, I think these topics are not related and buildQueryString function update will touch plenty of files

Reply:
Sure